examples/passage_reranker/config.yml

```diff
@@ -15,7 +18,71 @@ indexer:
     hosts: "localhost:9200"
     algorithm: "bm25"
 
+
```

Comment:
So we don't have to demonstrate the NER part here again, I guess. We have demonstrated in several parts that we can combine ranking + analysis already. Let's just focus on ranking in this case.